index.html

```diff
@@ -12,74 +12,235 @@
-                <li>Ruby On Rials</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-            </ul>
+    <!-- ----FIRST SECTION START-------->
+    <div class="first-section">
+        <header class="navbar">
+            <a href="#">
+                <h3 class="logo">Eg-Link</h3>
+            </a>
+            <nav>
+                <span class="material-symbols-outlined">
+                    menu
+                </span>
+            </nav>
+        </header>
+        <main>
+            <h1>Mostafa Shahat</h1>
+            <h2>Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </main>
+        <img class="hr" src="img/Vector-4.png" alt="hrl">
+        <section class="skils-container">
+            <div class="cont">
+                <span>language</span>
+                <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </div>
+            <div class="cont">
+                <span>Frameworks</span>
+                <ul class="list">
+                    <li>Bootstrap</li>
+                    <li>Ruby On Rials</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="cont">
+                <span>Skils</span>
+                <ul class="list">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
+        </section>
+        <img class="hr" src="img/Vector-4.png" alt="hrl">
+        <div class="social">
+            <a href="https://github.com/MostafaShahat2021" rel="noopener noreferrer nofollow" target="_blank">
+                <img src="/img/git.png" alt="GitHub-Profile">
+            </a>
+            <a href="https://www.linkedin.com/in/mostafa-shahat-a75810208/" rel="noopener noreferrer nofollow"
+                target="_blank">
+                <img src="/img/linkedin.png" alt="LinkedIn-profile">
+            </a>
+            <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                <img src="/img/Angelist.png" alt="">
+            </a>
+            <a href="https://twitter.com/Mostafa99738811" rel="noopener noreferrer nofollow" target="_blank">
+                <img src="img/Vectotr.png" alt="twitter">
+            </a>
+            <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                <img src="img/Vector1.png" alt="">
+            </a>
```

Comment:
- [ ] Kindly, complete the icons there are 5 in the Figma template, and make sure that they show in the browser. 👇🏽 
- [ ] Please 🙏🏽 use the `<ul>` tag like a container of the icons, for the best practices for HTML & CSS.

- Your design

![image](https://user-images.githubusercontent.com/58642949/186424598-5280ab6c-610d-4f55-b0ce-c67e6152d3ab.png)

- Figma template
![image](https://user-images.githubusercontent.com/58642949/186424702-7bc8c81e-6ef6-4d73-af4a-06a8838424c9.png)